install/source/index.markdown

```diff
@@ -49,7 +49,7 @@ By default we will assume you are building on the *latest* branch, we currently
 Pull down required repositories and build from within the root directory of your catkin workspace:
 
     wstool init src
-    wstool merge -t src https://raw.githubusercontent.com/ros-planning/moveit/melodic-devel/moveit.rosinstall
+    wstool merge -t src https://raw.githubusercontent.com/ros-planning/moveit/${ROS_DISTRO}-devel/moveit.rosinstall
```

Comment:
>  By default we will assume you are building on the *latest* branch, we currently use *melodic-devel* as our master branch. This branch builds for ROS Kinetic and newer, e.g. Ubuntu 16.04 and newer. If you would like to build an older release of MoveIt! from source, see the section below.

Reply:
I disagree with this change

Reply:
@davetcoleman This is a consistency change to match lines 54 and 55 which both reference the environment variable. Either these should all be `melodic` or all `${ROS_DISTRO}. Personally, I don't see the value of hardcoding the release name as it adds unnecessary maintenance overhead when we release future branches. Furthermore, referencing the environment variable makes this easier for users to copy-paste for releases other than melodic.

Reply:
Also, now that I read the quoted section again, I don't believe that it is correct. In my opinion, it should say "ROS **Melodic** and newer **on** Ubuntu 16.04 and newer." Even if the melodic-devel branch builds on ROS Kinetic for now (which I don't believe it does) it very likely will diverge heavily in the future

Reply:
You're missing the point and aren't following progress with MoveIt!

1) all new development should be on melodic branch, kinetic is stable
2) therefore all build-from source people should download melodic
3) melodic does build on kinetic, per our maintainer meetings, issues, and pull requests recently
4) the instructions guide people to do this. if you want to build kinetic from source, there are instructions at the bottom of the page for that